README.md

```diff
@@ -85,256 +79,48 @@ For more details, check out the rest of the [documentation](https://merlin.readt
-    (venv) $ merlin run workflows/feature_demo/feature_demo.yaml
-    # This launches workers that can process those tasks
-    (venv) $ merlin run-workers workflows/feature_demo/feature_demo.yaml
-
-
-# Using the CLI
-A good way to use merlin is through the command line interface (CLI).
-This allows you to both create tasks to be run, as well as stand up workers
-for those tasks.
-
-For more information see:
-
-    (venv) $ merlin --help
-    (venv) $ merlin run --help
-    (venv) $ merlin run-workers --help
-    (venv) $ merlin purge --help
-
-Run a workflow specified by the given file.
-
-`(venv) $ merlin run <my_workflow.yaml>`
-
-Stand up celery workers with queues specified in the workflow file.
-
-`(venv) $ merlin run-workers <my_workflow.yaml> [--echo] [--worker-args "celery args"] [--steps step1 stepN]`
-
-A note on arguments:
-
-`[--echo]` Just process the file and print the appropriate command.
-
-`[--worker-args "celery args"]` Passes arguments to the workers
-
-`[--steps step1 ... stepN]` Just give workers for specific steps in the file.
-
-
-To remove tasks from the task server, use the purge option:
-
-`(venv) $ merlin purge <my_workflow.yaml>`
-
-More information can be obtained by running:
-
-    (venv) $ merlin purge --help
-
-    usage: merlin purge [-h] [-f] [--steps PURGE_STEPS [PURGE_STEPS ...]]
-                        specification
-
-    positional arguments:
-      specification         Path to a Merlin YAML spec file
-
-    optional arguments:`
-      -h, --help            Show this help message and exit
-      -f, --force           Purge the tasks without confirmation (default: False)
-      --steps PURGE_STEPS [PURGE_STEPS ...]
-                            The specific steps in the YAML file from which you
-                            want to purge the queues. The input is a space
-                            separated list. (default: None)
-
-## Some real-life examples:
-
-Run workers for the feature_demo.yaml file:
-
-    (venv) $ merlin run-workers workflows/feature_demo/feature_demo.yaml --worker-args "--prefetch-multiplier 1"
-    $ celery worker -A merlin --prefetch-multiplier 1 -Q merlin,hello_queue,post_process_queue
-
-Just run workers for the hello step in the file:
-
-    (venv) $ merlin run-workers workflows/feature_demo/feature_demo.yaml --steps hello
-    $ celery worker -A merlin -Q hello_queue
-
-Adding `--echo` to a command will just print out the command, so you can move this command into more complex workflows.
-
-For instance, to put into a batch script, or run many workers you can do stuff like this:
-
-    $ srun -n 5 `merlin run-workers workflows/feature_demo/feature_demo.yaml --echo --steps hello`
-
-Which is equivalent to
-
-    $ srun -n 5 celery worker -A merlin -Q hello_queue
+namely a demo workflow that has simulation and machine-learning,
+first install the workflow's dependencies:
 
-Generate a template spec file.
+    $ pip install -r workflows/feature_demo/requirements.txt
```

Comment:
You're right, we need to set up `merlin-templates` so that users can use that along with the pip install.

As far as your suggestion, I don't see why not!

Reply:
See #71